package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "react-native start",
-    "test": "jest --runInBand",
+    "test": "jest --runInBand --silent",
```

Comment:
I think is better remove --silent, if you want to reduce the messages just change the loglevel